auth/auth-router.js

```diff
@@ -10,8 +10,8 @@ router.post('/register', (req, res) => {
   const hash = bcrypt.hashSync(user.password, 10); // 2 ^ n
   user.password = hash;
 
-  Users.findBy({ username: user.username }).first().then(user => {
-    if (user) {
+  Users.findBy({ username: user.username }).first().then(user1 => {
```

Comment:
I would suggest naming this something other than user1. A better name would be userFound, as that defines better what the variable is.